aiidalab_widgets_base/computational_resources.py

```diff
@@ -927,11 +927,20 @@ def on_setup_computer_success(self, function):
     def test(self, _=None):
         with self._test_out:
             clear_output()
-            print(
-                subprocess.check_output(
-                    ["verdi", "computer", "test", "--print-traceback", self.label.value]
-                ).decode("utf-8")
-            )
+            try:
+                print(
+                    subprocess.check_output(
+                        [
+                            "verdi",
+                            "computer",
+                            "test",
+                            "--print-traceback",
+                            self.label.value,
+                        ]
+                    ).decode("utf-8")
+                )
+            except subprocess.CalledProcessError as error:
+                print(error)
```

Comment:
Maybe we should return False here (and True otherwise) so that we can unit test this? In #502 I added a test for localhost setup, which should allow for testing a successful run of this function.

Reply:
done ✅